src/offset/fixed.rs

```diff
@@ -242,3 +242,68 @@ mod tests {
         );
     }
 }
+
+#[cfg(feature = "pyo3")]
+mod pyo3 {
+    use super::FixedOffset;
+    use pyo3::conversion::{IntoPy, ToPyObject};
+    use pyo3::types::PyDelta;
+
+    impl ToPyObject for FixedOffset {
+        fn to_object(&self, py: pyo3::Python) -> pyo3::PyObject {
+            let dt_module = py.import("datetime").expect("Failed to import datetime");
+            let dt_timezone = dt_module.getattr("timezone").expect("Failed to getattr timezone");
+            let seconds_offset = self.local_minus_utc();
+            let td =
+                PyDelta::new(py, 0, seconds_offset, 0, true).expect("Failed to contruct timedelta");
+            let offset = dt_timezone.call1((td,)).expect("Failed to call timezone with timedelta");
+            offset.into()
+        }
+    }
+
+    impl IntoPy<pyo3::PyObject> for FixedOffset {
+        fn into_py(self, py: pyo3::Python) -> pyo3::PyObject {
+            ToPyObject::to_object(&self, py)
+        }
+    }
+
+    // impl FromPyObject<'_> for FixedOffset {
+    //     fn extract(ob: &pyo3::PyAny) -> pyo3::PyResult<FixedOffset> {
+    //         // use python call directly until pyo3 supports zoneinfo
+    //         // not sure how since this needs a native utc type
+    //         let _date = <PyDate as PyTryFrom>::try_from(ob)?;
+    //         todo!("Not sure how since this needs a native utc type");
+    //         Ok(FixedOffset::east(0))
+    //     }
+    // }
+
+    #[test]
+    fn test_pyo3_topyobject() {
+        // use std::cmp::Ordering;
+
+        let gil = pyo3::Python::acquire_gil();
+        let py = gil.python();
+        let offset = FixedOffset::east(3600).to_object(py);
+        let py_module = py.import("datetime").unwrap();
+        let py_timezone = py_module.getattr("timezone").unwrap();
+        let py_timedelta = PyDelta::new(py, 0, 3600, 0, true).unwrap();
+        let py_timezone = py_timezone.call1((py_timedelta,)).unwrap();
+        // assert_eq!(offset.compare(py_timezone).unwrap(), Ordering::Equal);
+        // TODO: not sure how to convert this to timezone struct to
+        // compare since we don't really have a struct to downcast
+        assert_eq!(offset.as_ref(py), py_timezone);
```

Comment:
@davidtwco Do you know if there is any way that we are able to compare timezone? I tried comparing with `Eq` but even though both have the same value it didn't seemed to pass.